### PR TITLE
Add test id for kubevirt ready and available condition

### PR DESCRIPTION
**Pull request by petrkotas**
**What this PR does / why we need it**:
Add test for kubevirt ready and created condition.

**Special notes for your reviewer**:
Test was already present. Added simple test function to link the test to test id.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/operator_test.go [hunk 1]
```diff
@@ -657,7 +657,14 @@ spec:
 
 		// make sure virt deployments use shasums again after each test
 		ensureShasums()
-	})
+    })
+
+    Describe("[testid:1746]should provide created and ready condition", func() {
```
Comment (slintes) [inline tests/operator_test.go]: `ready` is `available` in the meanwhile

File: tests/operator_test.go [hunk 2]
```diff
@@ -657,7 +657,14 @@ spec:
...
+        kv := getCurrentKv()
+
+        By("Creating KubeVirt Object Created and Ready Condition")
```
Comment (slintes) [inline tests/operator_test.go]: Nothing is created here. You just verify conditions on the existing KubeVirt CR (which is ok, no need to create a new one). And ready -> available again.

File: tests/operator_test.go [hunk 3]
```diff
@@ -659,6 +659,13 @@ spec:
 		ensureShasums()
 	})
 
+	Describe("[testid:1746]should have created and ready condition", func() {
+		kv := getCurrentKv()
```
Comment (rmohr) [inline tests/operator_test.go]: This has to be in an `It`

Comment (slintes) [reply tests/operator_test.go]: ah, good catch
and again, there is no `ready` condition anymore, it was renamed to `available`... please change it at all places where you mention ready

**Pull request merged**